.circleci/config.yml

```diff
@@ -104,7 +104,9 @@ jobs:
           name: install linters
           command: |
             rm -rf $(stack path --dist-dir) $(stack path --local-install-root)
```

Comment:
I wonder what's the reason for this line, and whether it affects caching.

Reply:
You know, I'm not sure either(I just kept it from an old CI config). We can try removing it.
```suggestion
```

Reply:
(Commited the change through github UI)